agave_app/tfeditor/gradients.cpp

```diff
@@ -132,6 +151,79 @@ GradientEditor::GradientEditor(const Histogram& histogram, QWidget* parent)
   vbox->addWidget(m_customPlot);
 }
 
+void
+GradientEditor::changeEvent(QEvent* event)
+{
+  // this might be too many but ThemeChange only seems to work on the QMainWindow.
+  // At least on Windows, changing dark mode to light mode incurs StyleChange and PaletteChange events too.
+  if (event->type() == QEvent::ThemeChange || event->type() == QEvent::ApplicationPaletteChange ||
+      event->type() == QEvent::StyleChange || event->type() == QEvent::PaletteChange) {
+    // check for dark or light mode
+    auto sh = QGuiApplication::styleHints();
+    auto colorScheme = sh->colorScheme();
+    QColor plotLineColor;
+    QColor backgroundColor;
+    QColor barsColor;
+    QColor gridColor;
+    QColor subgridColor;
+    if (colorScheme == Qt::ColorScheme::Dark) {
+      barsColor = Qt::magenta;
+      barsColor.setAlphaF(0.5);
+      plotLineColor = this->palette().color(QPalette::Text);
+      backgroundColor = this->palette().color(QPalette::Window);
+      gridColor = plotLineColor.darker(150);
+      subgridColor = plotLineColor.darker(170);
+    } else if (colorScheme == Qt::ColorScheme::Light) {
+      barsColor = Qt::magenta;
+      barsColor.setAlphaF(0.25);
+      plotLineColor = this->palette().color(QPalette::Text);
+      backgroundColor = this->palette().color(QPalette::Window);
+      gridColor = plotLineColor.lighter(150);
+      subgridColor = plotLineColor.lighter(170);
+    }

```

Comment:
Nit: Some duplicated settings that could potentially be moved out, not a big deal though